### PR TITLE
Dashboard can be served on a non-root path

### PR DESCRIPTION
**Pull request by kazet**

### CODE REVIEW

**Review by psrok1 — CHANGES_REQUESTED** [review]
I believe there is more generic way than pasting base path everywhere e.g. blueprints + changing templates to accept another base path than `/`.

**Review by psrok1 — APPROVED** [review]
Looks OK! Tested manually with default and custom base path.

File: karton/dashboard/app.py [hunk 1]
```diff
@@ -43,6 +39,21 @@ class KartonDashboard(KartonBase):
 
 karton = KartonDashboard()
 
+base_path = (
+    karton.config.get("dashboard", "base_path")
+    if karton.config.has_option("dashboard", "base_path")
+    else ""
+)
```
Comment (msm-cert) [inline karton/dashboard/app.py:46]: Isnt this equivalent to just

```suggestion
base_path = karton.config.get("dashboard", "base_path") or ""
```
(note: i didn't test it in this case, but `None or "" == ""`)

**Review by msm-cert — APPROVED** [review]

File: karton/dashboard/app.py (hunk 1 above)
Comment (psrok1) [reply karton/dashboard/app.py:46]: Actually it's even more simple
```suggestion
base_path = karton.config.get("dashboard", "base_path", fallback="")
```
See Config.get definition https://github.com/CERT-Polska/karton/blob/master/karton/core/config.py#L87

**Comment by psrok1** [comment]
Wait... actually it should be minor bump, not bugfix :thinking:

**Pull request merged**